README.rst

```diff
@@ -195,6 +198,21 @@ you must build from a source distribution (see below).
 Binary wheels for other operating systems will be available in a future
 release.
 
+Windows
+-------
+
+Binary wheels for rasterio and GDAL are created by Christoph Gohlke and are available from his website.
+
+To install rasterio, simply download both binaries for your system (`rasterio
+<http://www.lfd.uci.edu/~gohlke/pythonlibs/#rasterio>`__ and `GDAL
+<http://www.lfd.uci.edu/~gohlke/pythonlibs/#gdal>`__) and run something like this from the downloads folder:
+
+.. code-block:: console
+
+    $ pip install wheel
```

Comment:
The wheel module is required to *build* wheels, but not required for installation of wheels. On the other hand, a `pip install -U pip` is likely required to get wheel support, so I suggest swapping that for `pip install wheel` above.